tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/MySql/MySqlConstants.cs

```diff
@@ -3,11 +3,13 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using Datadog.Trace.Configuration;
+
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.MySql
 {
     internal static class MySqlConstants
     {
-        public const string SqlCommandIntegrationName = "MySqlCommand";
+        public const string SqlCommandIntegrationName = nameof(IntegrationIds.AdoNet);
```

Comment:
If all of the `SqlCommandIntegrationName` values are the same, can we just remove it from `IAdoNetClientData` entirely and put the constant there?

Reply:
Yes, that makes sense. Will do

Reply:
done in 193c33a